torch/csrc/distributed/c10d/DispatchOps.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <c10/util/intrusive_ptr.h>
+#include <c10d/ProcessGroup.hpp>
+
+namespace c10d {
+
+// Belows are essentially ProcessGroup's corresponding ops but routed to the dispatcher.
+TORCH_API c10::intrusive_ptr<ProcessGroup::Work> dispatch_broadcast(const c10::intrusive_ptr<ProcessGroup>& process_group,
```

Comment:
Instead of calling this `dispatch_broadcast()` which does not really sound much descriptive, I would prefer to have a subnamespace `c10d::ops`, rename this back to `broadcast()` , and move into that namespace.

```
namespace c10d {
namespace ops {

TORCH_API c10::intrusive_ptr<ProcessGroup::Work> broadcast(...);

}
}
```

Reply:
I guess using namespace v.s. prefix_ is just a coding style preference. I feel like dispatch_ can really tell the users that this operation will go through the dispatcher.